tests/helix/send-to-helix-inner.proj

```diff
@@ -92,14 +92,17 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(NeedsWorkload)' == 'true'">
-    <HelixPreCommand Condition="'$(OS)' == 'Windows_NT'" Include="set PATH=%HELIX_CORRELATION_PAYLOAD%\dotnet-latest%3B%PATH%" />
-    <HelixPreCommand Condition="'$(OS)' != 'Windows_NT'" Include="export PATH=$HELIX_CORRELATION_PAYLOAD/dotnet-latest:$PATH" />
+    <!--<HelixPreCommand Condition="'$(OS)' == 'Windows_NT'" Include="set PATH=%HELIX_CORRELATION_PAYLOAD%\dotnet-latest%3B%PATH%" />-->
```

Comment:
@radical this too. Should we just remove this?

Reply:
I want to decide on what exactly needs to be removed, or retained in the templates9 PR, so I avoided removing stuff here.

Reply:
Sounds good, I'll leave it as-is then.